Base/usr/share/man/man7/kernel_boot_parameters.md

```diff
@@ -0,0 +1,142 @@
+## Name
+
+Kernel Boot Parameters - strings recognized by the kernel, that determine the
+functionality in runtime.
+
+## Description
+
+The kernel supports boot parameters to change some of its behavior. This
+functionality can be used to disable or enable features in the OS entirely, but
+can sometimes be used also to change a feature's behavior without disabling it
+entirely.
+
+Some of the boot parameters require a value after it. In that case, you will
+need to write them like this:
+
+```
+
+param1=value ...
+
+```
+For those that don't require, you should just write them without equals sign,
+like that:
+```
+
+param1 ...
+
+```
+
+## Where to put those parameters
+
+You can either add kernel boot parameters when you invoke ```./run```:
+
+```
+
+./run qcmd param1 param2 ...
+
+```
+
+for Q35 machines, use this layout:
+
+```
+
+./run q35_cmd param1 param2 ...
+
+```
+
+You may wish to write the kernel boot parameters in a more consistent method.
+For that, you can modify the grub configuration files. Since we have 3 of them
+(one for each partition scheme), you should pick the right one for you, so if
+you invoke for example:
+
+```
+
+./build-image-grub gpt
```

Comment:
In general, I think the manual should not explain the build process on the host OS. But then this is useful info. Hm...

Reply:
It's required to write about it, since without it I can't really explain what you should do in order to apply the cmdline modification in the GRUB configuration files.

Reply:
I could change it to something more "generic", and say that if you build a virtual disk with GPT partition scheme, you need to change the corresponding GRUB configuration file.

Reply:
Yeah that sounds better.